docs/en/connector-v2/source/Clickhouse.md

```diff
@@ -56,12 +55,6 @@ The query sql used to search data though Clickhouse server
 
 `ClickHouse` user password
 
-### schema [Config]
```

Comment:
Why delete `schema`?

Reply:
Clickhouse never support custom schema. It use schema projection.